README.md

```diff
@@ -69,7 +69,7 @@ and give your application a name and a description:
 At this point, the client ID and the client secret are available, so set those values to
 `spotify-oauth2-client-id` and `spotify-oauth2-client-secret`, respectively.
 
-Then, scroll down a little bit, type <http://localhost:8591/> as the Redirect
+Then, scroll down a little bit, type <http://localhost:8080/spotify-callback> as the Redirect
```

Comment:
If we decide to change the default port to `8591`, we'll need to change this as well.

```suggestion
Then, scroll down a little bit, type <http://localhost:8591/spotify-callback> as the Redirect
```